database/migrations/20210218_130252_create_session.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/beego/beego/v2/client/orm/migration"
+	"log"
+)
+
+// DO NOT MODIFY
+type CreateSession_20210218_130252 struct {
+	migration.Migration
+}
+
+// DO NOT MODIFY
+func init() {
+	m := &CreateSession_20210218_130252{}
+	m.Created = "20210218_130252"
+
+	err := migration.Register("CreateSession_20210218_130252", m)
+	if err != nil {
+		log.Fatal("Migration failed:", err)
+	}
+}
+
+// Run the migrations
+func (m *CreateSession_20210218_130252) Up() {
+	m.SQL(`CREATE TABLE "session"
+		(
+			session_key	char(64) NOT NULL,
+			session_data	bytea,
+			session_expiry	timestamp NOT NULL,
+			CONSTRAINT session_key PRIMARY KEY(session_key)
+		);`)
```

Comment:
The table structure here is from [the document](https://github.com/beego/beego/blob/develop/adapter/session/postgres/sess_postgresql.go) in `Beego` codebase, the table name also needs to be `session` which might not align with other existing table names